pyincore/utils/datasetutil.py

```diff
@@ -17,30 +17,38 @@
 
 class DatasetUtil:
     @staticmethod
-    def join_datasets(geodataset, tabledataset):
+    def join_datasets(geodataset, tabledataset, clean_attributes=True):
```

Comment:
could we default to "false" to be backwards compatible?

Reply:
changed to false